### ISSUE TITLE
Using renv

### ISSUE THREAD

**Issue by edward-burn** [mdlavallee92/CaprPhenotypePheburary#2]
Would it be possible to set up [renv](https://rstudio.github.io/renv/articles/renv.html) to go along with the RProj to help with reproducibility?

**Comment by ablack3** [comment]
Definitely! Feel free to add it to a PR or I'll add it in my next commit.

**Comment by mdlavallee92** [comment]
Good call @edward-burn, I have added an renv. 


The renv should have all the [`HADES`](https://ohdsi.github.io/Hades/index.html) ecosytem plus stuff from tidyverse. If there is anything missing in terms of packages please let me know.
